src/tasks/AutoBidAuctionTask.py

```
@@ -0,0 +1,401 @@
+import re
+from ok import TaskDisabledException, WaitFailedException
+from src.tasks.BaseNTETask import BaseNTETask
+
+class AutoBidAuctionTask(BaseNTETask):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.name = "自动拍卖"
+        self.description = "在拍卖主界面，选择低级会场后开始"
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)